cogs5e/initTracker.py

```diff
@@ -537,10 +537,11 @@ def wrapper(func):
 
         def mod_or_set(opt_name, old_value):
             new_value = args.last(opt_name, type_=int)
+            if old_value is None:
+                old_value = 0
```

Comment:
Nit: This would display, for example, `HP max set to 10 (was 0)` instead of `(was None)`.